_data/changelogs/docs-getting-started-devs-phase-3.yml

```diff
@@ -0,0 +1,76 @@
+title: "Phase 3: Customize"
+type: documentation
+changelogURL:
+items:
+  - date: 2022-06-17
+    summary: Replaced `_theme-settings.scss` with `_uswds-theme.scss`.
```

Comment:
I'm wondering if people will understand that this is a replacement to the text on the page rather than the actual file? See line 18 for an example of how I would expect this to read if it were just about text:

"Updated text to replace `_theme-settings.scss` with `_uswds-theme.scss`."

Reply:
Updated according to the suggestion in https://github.com/uswds/uswds-site/pull/1919#discussion_r1037584948

Updated in: https://github.com/uswds/uswds-site/pull/1919/commits/afe4b145041a1e2f999bf84ce6cdcf5c6c39f208